src/mgr/PyModule.cc

```diff
@@ -362,9 +362,13 @@ int PyModule::load(PyThreadState *pMainThreadState)
       const wchar_t *argv[] = {L"ceph-mgr"};
       PySys_SetArgv(1, (wchar_t**)argv);
       // Configure sys.path to include mgr_module_path
-      string paths = (g_conf().get_val<std::string>("mgr_module_path") + ':' +
+      std::filesystem::path mgr_module_path = g_conf().get_val<std::string>("mgr_module_path");
+      std::filesystem::path pypackages_path_absolute = mgr_module_path;
+      pypackages_path_absolute /= pypackages_path;
+      string paths = (mgr_module_path.string() + ":" + pypackages_path_absolute.string() + ':' +
                       get_site_packages() + ':');
       wstring sys_path(wstring(begin(paths), end(paths)) + Py_GetPath());
+      dout(1) << get_current_dir_name() << dendl;
```

Comment:
When adding log traces, plz add some context, like:
```suggestion
      dout(1) << "Current directory: '" get_current_dir_name() << "'" << dendl;
```
However, this looks more of a developer debug trace that can be removed from the final code, isn't it?

Reply:
it was intented to be removed before merge.